### ISSUE TITLE
Change the right-click gesture from a long press to a two-finger tap

### ISSUE THREAD

**Issue by zhengyongtao** [moonlight-stream/moonlight-android#1064]
The long-press behavior will make the PC think that the left mouse button is long-pressed, which makes us get wrong results. Can we change the right-click gesture from long-press to two-finger click?

**Comment by Daviid-P** [comment]
I usually tap and drag to where I want to right click, then while still pressing I tap with a different finger.

**Comment by zhengyongtao** [comment]
> 我通常点击并拖动到我想要右键单击的位置，然后在按住的同时用另一根手指点击。

I'm guessing you're using a touchpad, but I'm using a touchscreen

**Comment by Daviid-P** [comment]
> > 我通常点击并拖动到我想要右键单击的位置，然后在按住的同时用另一根手指点击。
> 
> I'm guessing you're using a touchpad, but I'm using a touchscreen

We're on moonlight-android so I'm also using a touchscreen

**Comment by zhengyongtao** [comment]
> > > 我通常点击并拖动到我想要右键单击的位置，然后在按住的同时用另一根手指点击。
> > 
> > 
> > I'm guessing you're using a touchpad, but I'm using a touchscreen
> 
> We're on moonlight-android so I'm also using a touchscreen
> 
> Edit: wait, you mean the mouse tracking mode? I do have it as "use touchscreen as a trackpad"

I use translation software, so there may be some problems with understanding. I mean settings -touch mode-touchscreen, are we talking about the same thing?

**Comment by zhengyongtao** [comment]
> > > 我通常点击并拖动到我想要右键单击的位置，然后在按住的同时用另一根手指点击。
> > 
> > 
> > I'm guessing you're using a touchpad, but I'm using a touchscreen
> 
> We're on moonlight-android so I'm also using a touchscreen
> 
> Edit: wait, you mean the mouse tracking mode? I do have it as "use touchscreen as a trackpad"

Yes, I think "use touchscreen as a trackpad" is right.

**Comment by Daviid-P** [comment]
> > > > 我通常点击并拖动到我想要右键单击的位置，然后在按住的同时用另一根手指点击。
> > > 
> > > 
> > > I'm guessing you're using a touchpad, but I'm using a touchscreen
> > 
> > 
> > We're on moonlight-android so I'm also using a touchscreen
> > Edit: wait, you mean the mouse tracking mode? I do have it as "use touchscreen as a trackpad"
> 
> Yes, I think "use touchscreen as a trackpad" is right.
Yeah, I believe it's the 6th setting or the 1st on "Input Settings" 
I have that ON.

**Comment by zhengyongtao** [comment]
> touchscreen

I mean when I use touch screen mode I can only use one finger long press to trigger the right mouse button, which makes windows think I'm long pressing the left mouse button. So currently I can only use touchpad mode to use the right button normally. I think it is possible to change the single-finger long-press behavior of touch screen mode to two-finger click

**Comment by Daviid-P** [comment]
Yes, I just tried and it seems it'd be better to have right click on a 2 fingers tap.

**Comment by RocoGreen** [comment]
Hi ! Can we add this feature please ^^

**Comment by timotheus0106** [comment]
yes pls, or make it changeable. i cant play a lot of strategy games and the long press right click gesture just dont let me play them smooth. in steamlink right click is with two fingers, which makes it a lot easier to use in games like civ, wastelans, two point university and so on.
